pages/common/transmission.md

```diff
@@ -0,0 +1,29 @@
+# transmission
+
+> Transmission is a simple torrent client.
+> Transmission isn't a command, but a set of commands. See the pages below.
+> More information: <https://transmissionbt.com/>.
+
+- Show the tldr page for `transmission-daemon`:
+
+`tldr transmission-daemon`
+
+- Show the tldr page for `transmission-remote`:
+
+`tldr transmission-remote`
+
+- Show the tldr page for `transmission-create`:
+
+`tldr transmission-create`
+
+- Show the tldr page for `transmission-edit`:
+
+`tldr transmission-edit`
+
+- Show the tldr page for `transmission-show`:
+
+`tldr transmission-show`
+
+- Show the tldr page for `transmission-cli` (depricated):
+
+`tldr transmission-cli`
```

Comment:
I'm unsatisfied with this format as it is, because it doesn't tell you anything about what each command does. I'd love some suggestions on how best to add more info. Here's an initial idea:

```suggestion
- Show the tldr page for running Transmission's daemon:

`tldr transmission-daemon`

- Show the tldr page for interacting with the daemon:

`tldr transmission-remote`

- Show the tldr page for creating torrent files:

`tldr transmission-create`

- Show the tldr page for modifying torrent files:

`tldr transmission-edit`

- Show the tldr page for getting info about torrent files:

`tldr transmission-show`

- Show the tldr page for the deprecated method for interacting with the daemon:

`tldr transmission-cli`
```

Thoughts?

Reply:
if `transmission-cli` is deprecated there is no requirement to document it.